### DISCUSSION TITLE
Adding function and calling feature on same button.

### DISCUSSION THREAD

**Discussion by IAM5K** [OfficeDev/microsoft-365-agents-toolkit#9058] [Q&A]
I am sending an adaptive card to the user and it has a button that can initiate a call with a person whose detail I can attach and pass to a button.
Parallelly, I want to send a trigger to the bot that the button has been clicked.
Is there a way I can implement this feature?

**Comment by therealjohn** [comment]
Hi @IAM5K, 

Can you share some snippet of how you're initiating the call with a person?
